plugins/assistant/api/assistantJobGeneral.js

```diff
@@ -5,6 +5,8 @@ const assistant = require("./assistant.js");
 const Parser = require('rss-parser');
 const underscore = require('underscore');
 const versionInfo = require('../../../frontend/express/version.info');
+const pluginManager = require('../../pluginManager.js');
+const offlineMode = pluginManager.getConfig("api").offline_mode;
```

Comment:
Also need to do local `getConfig` call instead of global one.
I am not sure if we need to loadConfigs here. You can ask @ArtursKadikis if this is loaded from api process, then no need, but if this is loaded from job, then you first need to use loadConfigs and only then getConfig inside the callback